config/kafka-monitor.properties

```diff
@@ -51,6 +52,16 @@
     }
   },
 
+  "topic-rebalancer": {
```

Comment:
To make the name consistent with other services, how about naming it "topic-rebalance-service"?